.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2.1
+
+orbs:
+  # Always take the latest version of the orb, this allows us to
+  # run specs against Solidus supported versions only without the need
+  # to change this configuration every time a Solidus version is released
+  # or goes EOL.
+  solidusio_extensions: solidusio/extensions@volatile
+
+jobs:
+  run-specs-with-postgres:
+    executor: solidusio_extensions/postgres
+    steps:
+      - checkout
+      - solidusio_extensions/run-tests-solidus-master
+      - solidusio_extensions/store-test-results
```

Comment:
This CI config is based on the default CI config used by Solidus extensions. However, I updated the config to test SolidusFrontend only against the master version of Solidus. The default CI config uses the `solidusio_extensions/run-tests` step which runs the tests against multiple Solidus versions. I believe there is no need to provide this level of test coverage for SolidusFrontend, since 1) we're deprecating SolidusFrontend, and 2) being originally part of Solidus, SolidusFrontend historically didn't provide backward compatibility to older Solidus versions.